MsixCore/msixmgr/CommandLineInterface.cpp

```diff
@@ -175,6 +175,19 @@ std::map<std::wstring, Options, CaseInsensitiveLess> CommandLineInterface::s_opt
                     return S_OK;
                 }),
             },
+            {
+                L"-output",
+                Option(true, IDS_STRING_HELP_OPTION_OUTPUT,
+                    [&](CommandLineInterface* commandLineInterface, const std::string& outPath)
+                {
+                    if (commandLineInterface->m_operationType != OperationType::Unpack)
+                    {
+                        return E_INVALIDARG;
+                    }
+                    commandLineInterface->m_outputPath = utf8_to_utf16(outPath);

```

Comment:
>m_outputPath [](http://example.com/codeflow?start=42&length=12)

Would rename this to m_unpackOutputPath as well.

Reply:
Actually is "output" set in stone? Without reading the help I'm not sure it would be obvious what the difference between "-output" and "-destination" would be. Maybe m_outputLoggingPath would be more clear at least in code if the cmdline interface arg name is set.

Reply:
Sure nice catch. Fixed!